src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -83,4 +103,12 @@ Item {
         width: positionMarker.size
         height: width
     }
+
+    ColorOverlay {
```

Comment:
This should gray the marker in case there is no accuracy/GPS position? Or what is the purpose? Can you please add comment?